ludwig/schema/model_config.py

```diff
@@ -128,7 +129,7 @@ class OutputFeaturesContainer(BaseFeatureContainer):
 
 @DeveloperAPI
 @ludwig_dataclass
-class ModelConfig(BaseMarshmallowConfig):
+class _ModelConfig(BaseMarshmallowConfig):
```

Comment:
Looks like we're not using this anymore, are we just keeping it around as a legacy reference?

Reply:
Yes, I'll add a comment. Idea is to remove in a follow-up PR, as this PR is already very big.